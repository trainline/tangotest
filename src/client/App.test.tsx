import React from 'react';
import { render } from '@testing-library/react';
import { shallow } from 'enzyme';

import App from './App';


describe('Enzyme test', () => {
  const wrapped = shallow(<App />);
  it('should render the App Component correctly', () => {   
    expect(wrapped).toMatchSnapshot();
  });
});

describe('RTL test', () => {
  it('renders learn react link', () => {
    const { getByText } = render(<App />);
    const linkElement = getByText(/learn react/i);
    expect(linkElement).toBeInTheDocument();
  });
})

