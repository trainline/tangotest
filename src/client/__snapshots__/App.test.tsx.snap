// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Enzyme test should render the App Component correctly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <p>
      Edit 
      <code>
        src/App.tsx
      </code>
       and save to reload.
    </p>
    <a
      className="App-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn React
    </a>
  </header>
</div>
`;
